### PR TITLE
Update spaceapi crate

### LINKED ISSUE #65: Move versions to extensions field
See https://github.com/coredump-ch/spaceapi-rs/pull/62

### PR DESCRIPTION
**Pull request by dbrgn**
- Update crate
- Move versions extension to extensions field
- Use StatusBuilder in tests

Fixes #65.

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -35,8 +35,8 @@ iron = "^0.5"
 urlencoded = "^0.5"
 router = "^0.5"
 error-type = "^0.1"
-serde_json = "^0.8"
-spaceapi = "^0.4.2"
+spaceapi = "^0.5.0"
+serde_json = "^1.0"
```
Comment (rnestler) [inline Cargo.toml]: Why did you change the order of the crates :smile:? Also maybe we should re-export the needed serde_json stuff from spaceapi? Otherwise we need to match the version manually here.

File: src/modifiers.rs [hunk 2]
```diff
@@ -36,21 +37,12 @@ pub struct LibraryVersions;
...
-
-        // Add to map
-        // TODO: Simplify this stuff once spaceapi-rs moved to serde
-        // and doesn't need Optional anymore.
-        if let Some(ref mut map) = status.ext_versions {
-            map.insert("spaceapi-rs".to_string(), api_version);
-            map.insert("spaceapi-server-rs".to_string(), server_version);
-        }
+        // Add to extensions
+        status.extensions.insert("versions".into(), Value::Object(versions));
```
Comment (rnestler) [inline src/modifiers.rs]: I just noticed: Why do we insert the version of the library *dynamically*? It should be pretty static data :wink:

Comment (dbrgn) [inline src/modifiers.rs]: Good point. Let's change that once we have the builder.

File: Cargo.toml (hunk 1 above)
Comment (dbrgn) [inline Cargo.toml]: Maybe, although I'm not sure if we should really do it.
